vendor/github.com/sshaman1101/grpccmd/gen.go

```diff
@@ -35,8 +35,10 @@ func (g *grpccmd) GenerateImports(file *generator.FileDescriptor) {
 	if len(file.GetService()) > 0 {
 		g.P("// grpccmd imports")
 		g.P(`import (
-	"github.com/sshaman1101/grpccmd"
+	"io"
+
 	"github.com/spf13/cobra"
+	"github.com/sshaman1101/grpccmd"
```

Comment:
wtf?

Reply:
This is not an import statement into this file, this line inserting into generated .go code.

Reply:
Oh, ok.